solutions/control-flow/predict-the-seat.js

```diff
@@ -0,0 +1,24 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, "utf-8");
+let idx = 0;
+data = data.split("\n");
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+let t = parseInt(readLine());
```

Comment:
rename "t" as "numOfTestCases" and it can be on "const" type since we are not changing it